sdk/src/linux/usb_device_linux.cpp

```diff
@@ -673,3 +689,47 @@ aditof::Status UsbDevice::readLaserTemp(float &temperature) {
 
     return Status::OK;
 }
+
+aditof::Status UsbDevice::setCalibrationParams(const std::string &mode,
```

Comment:
The implementations are very similar on all platforms. Group code that is duplicate into some utils functions or a helper object.

Reply:
Moved the two methods that compute and apply the calibration data in a Utils class.